test/Microsoft.AspNet.Razor.Test/TagHelpers/TagHelperParseTreeRewriterTest.cs

```diff
@@ -923,7 +923,8 @@ public static TheoryData PrefixedTagHelperBoundData
                             new TagHelperAttributeDescriptor(
                                 name: "bound",
                                 propertyName: "Bound",
-                                typeName: typeof(bool).FullName),
+                                typeName: typeof(bool).FullName,
+                                isStringProperty: false),
```

Comment:
No need to specify `isStringProperty` for these tests, it'll pull from the `typeName` now